po/fr.po

```diff
@@ -0,0 +1,178 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-03-07 13:10+0100\n"
+"PO-Revision-Date: 2020-03-10 20:27+0000\n"
+"Last-Translator: Anne017 <anneonyme017@openmailbox.org>\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: qml/AddPage.qml:42
+msgid "Url (e.g. https://example.com)"
+msgstr "Lien (ex : https://exemple.com)"
```

Comment:
```suggestion
msgstr "Lien (ex : https://example.com)"
```
Let's keep https://example.com here, since this site actually exists and is safe to use for demonstration purposes. I should probably make the url untranslatable : )